### ISSUE TITLE
Check if `javac` is installed

### ISSUE THREAD

**Issue by volodya-lombrozo** [yegor256/jhome#20]
Currently we just throw exception if `javac` is not installed. It would we nice to have other option to check that `javac` is installed, for example by using method `boolean hasJavac()` or similar. Otherwise we have the only option to check presence of `javac` by catching an exception, which is an [anti-pattern](https://softwareengineering.stackexchange.com/questions/189222/are-exceptions-as-control-flow-considered-a-serious-antipattern-if-so-why) in fact.

**Comment by yegor256** [comment]
@volodya-lombrozo what's wrong with `new Jhome().javac().toFile().exists()`?

**Comment by yegor256** [comment]
@volodya-lombrozo yes, let's do it: `javacExists()` and `javaExists()` would be reasonable to have (similar to `exists()` for Java `File`)

**Comment by volodya-lombrozo** [comment]
@yegor256 I don't think we have to check `java` existence (since if we run java code, it implies that we have java installed)
